server/services/licenceService.ts

```diff
@@ -38,6 +42,22 @@ export interface LicenceRecord {
   approvedVersionDetails: ApprovedVersionDetails
 }
 
+export function adaptFieldConfigToSelectWorkingAddress(existingLicence, fieldConfigs) {
+  const curfewAddressPath = pickCurfewAddressPath(selectPathsFrom(existingLicence))
+
+  const modifiedFieldConfigs = R.clone(fieldConfigs).map((fieldConfig) => {
+    const fieldName = Object.keys(fieldConfig)[0]
+    const { licencePosition } = fieldConfig[fieldName]
+    if (licencePosition[0] === 'proposedAddress' && licencePosition[1] === 'curfewAddress') {
```

Comment:
guessing the elements in these don't refer to any thing other than positions in a path? (we cant extract them out and give them a better name?)